core/domain/suggestion_registry.py

```diff
@@ -1481,6 +1481,211 @@ def to_dict(self):
         }
 
 
+class TranslationReviewStats:
+    """Domain object for the TranslationReviewStatsModel."""
+
+    def __init__(
+            self, language_code, contributor_user_id, topic_id,
+            reviewed_translations_count, reviewed_translation_word_count,
+            accepted_translations_count,
+            accepted_translations_with_reviewer_edits_count,
+            first_contribution_date, last_contribution_date):
+        self.language_code = language_code
+        self.contributor_user_id = contributor_user_id
+        self.topic_id = topic_id
+        self.reviewed_translations_count = reviewed_translations_count
+        self.reviewed_translation_word_count = reviewed_translation_word_count
+        self.accepted_translations_count = accepted_translations_count
+        self.accepted_translations_with_reviewer_edits_count = (
```

Comment:
Re https://github.com/oppia/oppia/pull/15845#discussion_r939568257: Just double-checking—`accepted_translations_with_reviewer_edits_word_count` is no longer being used to calculate any internal scores either? And I don't see `accepted_translations_with_reviewer_edits_count` in the stats mocks either, so where will it be used?

Reply:
`accepted_translations_with_reviewer_edits_count` is required to generate [badges for corrections](https://www.figma.com/file/SNOHAatFcCvdvIAdxAzF6E/Contributor-Dashboard-Achievements?node-id=789%3A664). In case if we decide to drop badges based on the feedback for gamification we may drop this. Until we can keep it.

Reply:
I see thanks.